.bash_aliases

```diff
@@ -50,27 +56,41 @@ function ls-lined-executables() { ls-lined | grep -iE '[*]$' ; }
 function ls-lined-executables-c() { ls-lined-executables | xargs -I {} bash -c 'colorize Green "{}"' ; }
 alias lsc="ls-lined-folders-c && ls-lined-files-c && ls-lined-executables-c"
 
+# VS Code
+alias edit_snippets="$EDIT ~/Library/Application\ Support/Code/User/Snippets/*"
```

Comment:
This needs to be fixed before merging PR!! Needs to be:
```
alias edit_snippets="open ~/Library/Application\ Support/Code/User/Snippets"
```
I'll fix it in a moment! :)

Reply:
Fixed in follow-up commit, ID: d4673449652605f58616c3e6cfe54a6be85e8b24